src/components/listItemComponent/ListItemComponent.jsx

```diff
@@ -0,0 +1,34 @@
+import { useSelector } from "react-redux";
+import { selectRedditReturn } from '../reddit/RedditSlice'
+
+const ListItemComponent = (props) => {
+    const redditApiReturn = useSelector(selectRedditReturn);
+    console.log('redditApiReturn', redditApiReturn)
+
+    if(!redditApiReturn.data) {
+        console.log('api return is not true')
+        return <div>reddit is loading...</div>
+    } else 
+        return (
+            <div>
+                {
+                    redditApiReturn.data.children.map((each, index)=> {
+                        return (
+                            <div key={index}>
+                                <div style={{width: '1000px', height: '100px', backgroundColor: 'white', margin: '10px', key: '{generatedKey}'}}>
```

Comment:
should be a styled component with this many inline styles, but you can leave this for now. We'll work on styling these list items on the next ticket

Reply:
👍🏻